sycl/source/detail/accessor_impl.cpp

```diff
@@ -22,6 +23,15 @@ AccessorImplHost::~AccessorImplHost() {
   }
 }
 
+void AccessorImplHost::resize(size_t GlobalSize) {
+  if (GlobalSize != 1) {
```

Comment:
Why 1 is special value here?

Reply:
No special significance anymore, but now it prevents some unnecessary operations.

Reply:
OK, then, when do you call `resize` method with `GlobalSize` equal to one?